src/Loggers/MassTransit.SerilogIntegration/SerilogLog.cs

```diff
@@ -0,0 +1,269 @@
+﻿// Copyright 2007-2016 Chris Patterson, Dru Sellers, Travis Smith, et. al.
+//  
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the 
+// License at 
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0 
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR 
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the 
+// specific language governing permissions and limitations under the License.
+namespace MassTransit.SeriLogIntegration
+{
+    using System;
+    using System.Collections.Generic;
+    using Logging;
+    using Serilog.Events;
+
+
+    public class SerilogLog : ILog
+    {
+        public const string ObjectLogTemplate = "{@obj}";

```

Comment:
Thanks, I pulled the code directly from the original GitHub repository, and only made the build work. I'll tweak the casing of the object, as you suggest before committing.